### DISCUSSION TITLE
Table: how to add rows for "group total" and "sum total"?

### DISCUSSION THREAD

**Discussion by workflowsguy** [Textualize/rich#2667] [Q&A]
For a table containing groups of items, I need to add multiple rows showing "group total" values and a "sum total".

This feature is shown in the demonstration video created by [https://github.com/Textualize/rich/blob/master/examples/table_movie.py](https://github.com/Textualize/rich/blob/master/examples/table_movie.py), but I could not find any documentation how to do this (like what to observe when data attributes in a column are left aligned but the "Total" text is right-aligned.)

As there is no explanation in the sample code itself and trying to step through the code to understand it, it  also did not help me as the Rich output does not render dynamically in PyCharm.

Any explanations are appreciated!

**Comment by willmcgugan** [comment]
The totals in the table move are simply inserted as rows. As a general rule, the Rich Table class will render things, but it won't manipulate your data.
